src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -44,18 +46,38 @@ public Cohort()
         public string AccountLegalEntityPublicHashedId { get; set; }
         public Originator Originator { get; set; }
 
-        public virtual ICollection<Apprenticeship> Apprenticeship { get; set; }
-        public virtual ICollection<Message> Message { get; set; }
-        public virtual ICollection<TransferRequest> TransferRequest { get; set; }
+        public virtual ICollection<Apprenticeship> Apprenticeships { get; set; }
+        public virtual ICollection<Message> Messages { get; set; }
+        public virtual ICollection<TransferRequest> TransferRequests { get; set; }
 
-        public virtual void AddDraftApprenticeship(DraftApprenticeshipDetails draftApprenticeshipDetails)
+        [NotMapped]
```

Comment:
Could this be declared in `CohortConfiguration` rather than adding attributes to the model?